src/main/duke/src/Deadline.java

```diff
@@ -0,0 +1,31 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+/**
+ * Contains Deadlines, a subclass of Task
+ */
+public class Deadline extends Task {
+    private final String taskType = "[D]";
+    private LocalDate byTime;
+
+    /**
+     * Constructor for Deadline
+     * @param String description of deadline
+     * @param String by, which is a LocalDate
+     */
+    public Deadline(String description, String by) {
+        super(description);
+        this.byTime = LocalDate.parse(by);
```

Comment:
Could have a more descriptive name for `LocalDate`  like perhaps `LocalDateParser` for easier interpretation of this static method.